portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsAccessibility.testcase

```diff
@@ -0,0 +1,106 @@
+@component-name = "portal-web-forms-and-data-lists"
+definition {
+
+	property osgi.module.configuration.file.names = "com.liferay.frontend.js.loader.modules.extender.internal.configuration.Details.config";
+	property osgi.module.configurations = "waitTimeout=\"15\"";
```

Comment:
Test looks great! But do we need these? @davisaints

Reply:
Removed!